makefiles/Makefile.inc

```
@@ -29,6 +29,11 @@
 
 # Backwards-compatibility wrapper for people using
 # "include $COCOTB/makefiles/Makefile.inc" in their Makefile.
+
+$(warning 'Deprecated cocotb file structure detected. Please install cocotb \
+(see documentation) and set "COCOTB:=$$(shell cocotb-config --share)" \
```

Should this say:

```suggestion
(see documentation) and set "COCOTB_SHARE_DIR :=$$(shell cocotb-config --share)" \
```
Or is this advice now just wrong and can be removed?

---

As I know in the past we used `COCOTB` everyware (still used in tests). I think advice should stay and we may remove those Makefiles at some point.

---

Can you give an example of where we use / used to use the `$COCOTB` environment variable?

---

https://github.com/cocotb/cocotb/blob/master/tests/designs/sample_module/Makefile#L54

See v1.1 docs: https://cocotb.readthedocs.io/en/v1.1/quickstart.html#creating-a-makefile

---

Thanks!